server/game/cards/02_SHD/units/GreySquadronYWing.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer } from '../../../core/Constants';
+
+export default class GreySquadronYWing extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '9472541076',
+            internalName: 'grey-squadron-ywing'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addOnAttackAbility({
+            title: 'An opponent chooses a unit or base they control. You may deal 2 damage to it',
+            targetResolver: {
+                choosingPlayer: RelativePlayer.Opponent,
```

Comment:
Just leaving a placeholder comment here from our discussion in chat

Reply:
y wing was deleted